tests/test_fetch_same.py

```diff
@@ -0,0 +1,65 @@
+from nose.tools import assert_true, assert_false, assert_equal, \
+                        assert_list_equal, raises
+from . import PREFIX, CONN_INFO
+import datajoint as dj
+# import importlib
+# dj = importlib.import_module('datajoint-python.datajoint', None)
+import numpy as np
+
+Schema = dj.schema(PREFIX + '_fetch_same', connection=dj.conn(**CONN_INFO))
```

Comment:
Our conventions is to keep the `schema` object in lower case. For example `dj.Diagram` will look for the `schema` object in the module by that name.

Reply:
Updated.